rpmbuild/SPECS/R_packages-3.4.0-gcb01.spec

```diff
@@ -0,0 +1,709 @@
+# aci-ref-app-category:Programming Tools; aci-ref-app-tag:Compiler
+%define apptags aci-ref-app-category:Programming Tools; aci-ref-app-tag:Interpreter
+%define apppublication %{nil}
+
+
+#
+# enter a description, often a paragraph; unless you prefix lines with spaces,
+# rpm will format it, so no need to worry about the wrapping
+#
+%description
+This is the set of most popular packages added on to R that originate from CRAN.
+It's meant to be paired with an R_core module.
+
+
+
+#------------------- %%prep (~ tar xvf) ---------------------------------------
+
+%prep
+
+
+#
+# unpack the sources here.  The default below is for standard, GNU-toolchain
+# style things -- hopefully it'll just work as-is.
+#
+
+umask 022
+cd "$FASRCSW_DEV"/rpmbuild/BUILD
+rm -rf %{name}-%{version}
+mkdir %{name}-%{version}
+cd %{name}-%{version}
+
+
+#--- name all the packages to install
+#
+# this is the union of:
+#	* what's in our old centos6/R-3.0.2 (most popular R module)
+#	* what's in our old math/R-3.0.1 (2nd most popular R module)
+#	* top 100 most popular packages by downloads for part of 2013:
+#	  http://www.r-statistics.com/2013/06/top-100-r-packages-for-2013-jan-may/
+#
+# minus built-in stuff
+#	KernSmooth
+#	MASS
+#	Matrix
+#	base
+#	boot
+#	class
+#	cluster
+#	codetools
+#	compiler
+#	datasets
+#	foreign
+#	grDevices
+#	graphics
+#	grid
+#	lattice
+#	methods
+#	mgcv
+#	nlme
+#	nnet
+#	parallel
+#	rpart
+#	spatial
+#	splines
+#	stats
+#	stats4
+#	survival
+#	tcltk
+#	tools
+#	utils
+#
+# minus these that failed (grepping output for `ERROR'):
+#	RODBC
+#	RWeka
+#	RWekajars
+#	RcppGSL
+#	Rmpi
+#	WGCNA
+#	XLConnect
+#	copula
+#	diversitree
+#	gsl
+#	impute
+#	isva
+#	ncdf
+#	qvalue
+#	rJava
+#	rgdal
+#	topicmodels
+#	xlsx
+#	xlsxjars
+#
+# minus these that ended up not building (e.g. not in CRAN)
+#	AnnotationDbi
+#	BSgenome
+#	BayesTree
+#	Biobase
+#	BiocGenerics
+#	BiocInstaller
+#	Biostrings
+#	DESeq
+#	DiffBind
+#	GenomicFeatures
+#	GenomicRanges
+#	Gviz
+#	IRanges
+#	IlluminaHumanMethylation450kanno.ilmn12.hg19
+#	Models
+#	Proportional-Odds
+#	Rsamtools
+#	ShortRead
+#	Snowball
+#	XVector
+#	annotate
+#	biomaRt
+#	biovizBase
+#	bumphunter
+#	ctc
+#	cummeRbund
+#	edgeR
+#	genefilter
+#	geneplotter
+#	genomes
+#	hopach
+#	illuminaio
+#	int64
+#	limma
+#	manipulate
+#	marray
+#	methylumi
+#	minfi
+#	multtest
+#	pcaMethods
+#	peer
+#	preprocessCore
+#	rstan
+#	rstudio
+#	rtracklayer
+#	siggenes
+#	slider
+#	spp
+#	zlibbioc
+#
+# and these packages show up, too (pulled in a dependencies I guess)
+#	DEoptimR
+#	Defaults
+#	NLP
+#	TH.data
+#	expm
+#	fastmatch
+#	fracdiff
+#	gss
+#	highr
+#	httr
+#	mime
+#	permute
+#	tcltk2
+#	whisker
+
+cat > package_list <<EOF
+stringi
+ADGofTest
+AnDE
+BB
+Brobdingnag
+CpGassoc
+DBI
+FNN
+Formula
+GSA
+GWAF
+GenABEL
+GenABEL.data
+Hmisc
+MCMCpack
+MatchIt
+MetaSKAT
+R.methodsS3
+R2WinBUGS
+RColorBrewer
+RCurl
+RJSONIO
+RSQLite
+RSQLite.extfuns
+RSiena
+Rcmdr
+Rcpp
+RcppArmadillo
+RcppEigen
+Rmpi
```

Comment:
`Rmpi` is present here in R_packages. I'm assuming it doesn't work, should we remove it from the module?

Reply:
These other spec files were not committed to this repository even though they have been deployed to HARDAC. I didn't create these files. I am just adding them to the helmod repo since the new Rmpi depends upon them. 

My understanding is if you should not change a deployed spec file. The standard approach is to add an another spec file named `R_packages-3.4.0-gcb02.spec` and make the change there. 

There is a note further down in this spec file about `#NOTE: some may fail`.

Reply:
@dleehr Created issue #7 to address this.